client/blocks/image-editor/index.jsx

```diff
@@ -86,6 +88,10 @@ const ImageEditor = React.createClass( {
 			media: currentMedia
 		} = this.props;
 
+		if (newProps.isEscPressed) {
```

Comment:
Need spaces between parens ([reference](https://github.com/Automattic/wp-calypso/blob/2490b4cb4f22ee87fd050e14124137cd42d4d381/docs/coding-guidelines/javascript.md#examples-of-good-spacing)).